packages/algob/src/types.ts

```diff
@@ -413,22 +451,12 @@ export interface ASADeploymentFlags extends TxParams {
   creator: Account
 }
 
-export enum ASC1Mode {
-  DELEGATED_APPROVAL,
-  CONTRACT_ACCOUNT
-}
-
 export interface FundASCFlags {
   funder: Account
   fundingMicroAlgo: number
 }
 
-export interface SSCDeploymentFlags {
-  sender: Account
-  localInts: number
-  localBytes: number
-  globalInts: number
-  globalBytes: number
+export interface SSCOptionalFlags {
   appArgs?: Uint8Array[]
   accounts?: string
```

Comment:
this should be an array of strings. ()[https://developer.algorand.org/docs/features/asc1/stateful/#write-to-state]

Reply:
https://developer.algorand.org/docs/reference/transactions/#application-call-transaction
It says list of accounts, but type is mentioned as `address`.
what do you think ?

Reply:
let's change it to array , there may be error

Reply:
I think it should be an array. I checked `js-algorand-sdk`'s `makeApplicationNoOpTxn(..)`